app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/import/SettingsImportFragment.kt

```diff
@@ -28,6 +29,12 @@ class SettingsImportFragment : Fragment() {
     private lateinit var settingsImportAdapter: FastAdapter<ImportListItem<*>>
     private lateinit var itemAdapter: ItemAdapter<ImportListItem<*>>
 
+    private val pickDocumentReceiver = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
```

Comment:
Please revert the changes to this file. They are now unrelated to the bug fix. 
If you still wish to switch to `ActivityResultContracts`, please create a new pull request for that.